src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -84,7 +86,9 @@ public DocumentBuildContext(string buildOutputFolder, IEnumerable<FileAndType> a
 
         public ImmutableArray<string> ExternalReferencePackages { get; }
 
-        public ImmutableArray<string> XRefMapUrls { get; }
+        private ImmutableArray<string> XRefMapUrls { get; }
```

Comment:
ok

---
In reply to: [128892819](https://github.com/dotnet/docfx/pull/1837#discussion_r128892819) [](ancestors = 128892819)